src/main/java/com/blackduck/integration/detect/configuration/DetectProperties.java

```diff
@@ -1224,6 +1224,13 @@ private DetectProperties() {
             .setGroups(DetectGroup.NUGET, DetectGroup.GLOBAL, DetectGroup.SOURCE_SCAN)
             .build();
 
+    public static final NullablePathProperty DETECT_NUGET_ARTIFACTS_PATH =
+            NullablePathProperty.newBuilder("detect.nuget.artifacts.path")
+                    .setInfo("Nuget Artifacts Path", DetectPropertyFromVersion.VERSION_10_3_0)
+                    .setHelp("The path to the build artifacts of the nuget project if not stored on regular path. i.e the path to the output of obj directory if not stored on the default path.")
```

Comment:
Suggestion for condensing the help message. (Which hopefully still makes sense. ; ) )

"The path to the obj directory build artifacts of the NuGet project, if not the default path."